go-controller/pkg/ovn/pods_test.go

```diff
@@ -456,7 +457,7 @@ var _ = Describe("OVN Pod Operations", func() {
 				Expect(err).NotTo(HaveOccurred())
 				Eventually(func() string { return getPodAnnotations(fakeOvn.fakeClient.KubeClient, t.namespace, t.podName) }, 2).Should(MatchJSON(podJSON))
 
-				v4AddressSetName := t.namespace + ipv4AddressSetSuffix
+				v4AddressSetName := t.namespace + addressset.GetIPv4AddressSetSuffix()
```

Comment:
`GetIPv4ASName(t.namespace)`

Reply:
changed the code so this is not required